bit_stream_filter_context_test.go

```diff
@@ -0,0 +1,42 @@
+package astiav
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestBitStreamFilterContext(t *testing.T) {
+	bsf := FindBitStreamFilterByName("null")
+	require.NotNil(t, bsf)
+
+	bsfc, err := AllocBitStreamFilterContext(bsf)
+	require.NotNil(t, bsfc)
+	require.Nil(t, err)
+	defer bsfc.Free()
+
+	cl := bsfc.Class()
+	require.NotNil(t, cl)
+	require.Equal(t, "AVBSFContext", cl.Name())
+
+	bsfc.SetTimeBaseIn(NewRational(15, 1))
+	require.Equal(t, NewRational(15, 1), bsfc.TimeBaseIn())
+
+	fc, err := globalHelper.inputFormatContext("video.mp4")
+	require.NoError(t, err)
+	ss := fc.Streams()
+	require.Len(t, ss, 2)
+	s1 := ss[0]
+
+	cp1 := s1.CodecParameters()
```

Comment:
Instead of using `s1.CodecParameters()` could you alloc a new codec parameter (don't forget to free it) whom you'll can only set the codec id in order to test `CodecParameterIn` setter and getter?